ee/vellum_cli/pull.py

```diff
@@ -218,6 +218,10 @@ def pull_command(
         raise Exception(
             "The API we tried to pull from returned an invalid zip file. Please make sure your `VELLUM_API_URL` environment variable is set correctly."  # noqa: E501
         )
+    except ApiError:
+        raise Exception(
+            "The API we tried to pull is invalid. Please make sure your `VELLUM_API_URL` environment variable is set correctly."  # noqa: E501
+        )
```

Comment:
[oos] same feedback as `ping`. Could we detect 401 and warn about api key in that case?